.teamcity/test/VCSTest.kt

```diff
@@ -0,0 +1,15 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+import org.junit.Test
+import kotlin.test.assertTrue
+
+class VCSTest {
+    @Test
+    fun testSomeRootsHasAuthMethodWithPassword() {
```

Comment:
Minor thing but I'd avoid test prefix in tests. I'm not really a fan of test suffix in classes, but I understand for discoverability they may be useful.